puppetserver/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG UBUNTU_CODENAME=jammy
 
 FROM ubuntu:22.04 as base
 
-ARG PACKAGES=ca-certificates\ git\ netbase\ openjdk-17-jre-headless\ ruby3.0
+ARG PACKAGES="ca-certificates git netbase openjdk-17-jre-headless ruby3.0 ruby3.0-dev openssh-client gcc make cmake pkg-config libssl-dev libc6-dev"
```

Comment:
can we / should we uninstall the dev packages after the gems compiled their extensions?